flytekit/types/structured/structured_dataset.py

```diff
@@ -762,17 +759,9 @@ def _get_dataset_type(self, t: typing.Union[Type[StructuredDataset], typing.Any]
         # Get the column information
         converted_cols = self._convert_ordered_dict_of_columns_to_list(column_map)
 
-        # Get the format
-        default_format = (
-            original_python_type.DEFAULT_FILE_FORMAT
-            if issubclass(original_python_type, StructuredDataset)
-            else self.DEFAULT_FORMATS.get(original_python_type, PARQUET)
```

Comment:
shouldn't we still keep this?  The extract_cols_and_format call above, which returns the `storage_format` variable, will only look at Python `Annotate` annotations.  If the function signature specifies only a naked dataframe like (eg `pd.DataFrame`) wouldn't this now return `""`, even if there is a default format specified?